.drone.yml

```diff
@@ -2,6 +2,10 @@ workspace:
   base: /go
 
 pipeline:
+  clone:
+    image: plugins/git
+    tags: true
```

Comment:
So I don't know why you need to set new environment `TAG`, you can just use `${DRONE_TAG}` which drone already supported.